src/pageObject.js

```diff
@@ -1,16 +1,44 @@
 'use strict';
 
 class Header {
+  constructor(logo = 'Conduit logo') {
+    this.logo = logo;
+  }
 
-};
+  getLogo() {
+    return this.logo;
+  }
+}
 
 class PageObject {
+  constructor(url, header = new Header()) {
+    this.url = url;
+    this.header = header;
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.getLogo()}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug, header = new Header()) {
```

Comment:
Same issue here with the `header` default parameter. Better to create a new `Header` instance inside the constructor if none is provided.